### PR TITLE
feat(swagger): Serve up swagger docs

### LINKED ISSUE #86: Enable Swagger Docs
We should publish an OpenAPI spec and Swagger Docs.

### PR DESCRIPTION
**Pull request by samoehlert**
Closes #86 

Another method for understanding the API

### CODE REVIEW

File: scram/route_manager/tests/test_swagger.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import pytest
+from django.urls import reverse
+
+
+@pytest.mark.django_db
+def test_swagger_api(client):
+    """Test that the Swagger API endpoint returns a successful response."""
+    url = reverse("swagger-ui")
+    response = client.get(url)
```
Comment (grigorescu) [inline scram/route_manager/tests/test_swagger.py:11]: Can you check that the response contains something to ensure that swagger loaded the API? e.g. `"Entry"`

**Review by grigorescu — CHANGES_REQUESTED** [review]
One small thing. I've been thinking that we should have a test for every package we depend on, so it's trivial to know if an upgrade is ok or not, so great to see the tests in here.

File: scram/route_manager/tests/test_swagger.py (hunk 1 above)
Comment (samoehlert) [reply scram/route_manager/tests/test_swagger.py:11]: My first attempts are big fails. It looks like the client response only holds the HTML and the redoc and swagger stuff is all done via JS. That starts to get into selenium type territory which I'd rather not do.

File: scram/route_manager/tests/test_swagger.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_schema_api(client):
+    """Test that the Schema API endpoint returns a successful response."""
+    url = reverse("schema")
+    response = client.get(url)
+    assert response.status_code == 200
```
[the remark this answers is not in the document]
Comment (samoehlert) [reply scram/route_manager/tests/test_swagger.py:28]: Ok added to this test to make sure we are seeing the actiontypes, entries, ignore_entries, and uesrs listed.

**Review by grigorescu — APPROVED** [review]

**Pull request merged**